File: Assets/scenes/landinglaunchscene/script/RingRotationBehaviour.js
#pragma strict

var speed : float = 1.0f;

private var rot : Quaternion;
function Start () {
	rot = transform.rotation;
}

function Update () {

}

function FixedUpdate(){
	rot = Quaternion.Euler(speed, 0, 0);
	transform.rotation *= rot;
}

File: Assets/scenes/landinglaunchscene/script/RingRotationBehaviour.cs
using UnityEngine;
using System;


public class RingRotationBehaviour:MonoBehaviour{
	
	public float speed = 1.0f;
	
	Quaternion rot;
	public void Start() {
		rot = transform.rotation;
	}
	
	public void Update() {
	
	}
	
	public void FixedUpdate(){
		rot = Quaternion.Euler(speed, 0.0f, 0.0f);
		transform.rotation *= rot;
	}

}